src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SSPI/SSPIContextProvider.cs

```diff
@@ -7,7 +7,8 @@
 
 namespace Microsoft.Data.SqlClient
 {
-    internal abstract class SSPIContextProvider
+    /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SSPIContextProvider.xml' path='docs/members[@name="SSPIContextProvider"]/SSPIContextProvider/*'/>
+    public abstract class SSPIContextProvider
```

Comment:
If we're making this public, I'd really like to change this to "SspiContextProvider" to follow C# naming conventions. I won't demand it be changed all over the codebase, since that would drastically increase the scope of this PR. But since we won't be able to change it easily after we make it public, and I've been slowly fixing naming convention issues like this, we should take this opportunity to fix a long standing naming issue.

Reply:
Done in #2454 and carried through here now